sdl2-vga-terminal/src/VgaTerminal.cpp

```diff
@@ -78,27 +85,47 @@ VgaTerminal::VgaTerminal(const std::string &title, const int width, const int he
     }
 }
 
-void VgaTerminal::renderChar(const SDL_Point& dst, const uint8_t col, const uint8_t bgCol, const char c)
+void VgaTerminal::_renderCharLine(const std::bitset<8> line, const int dstx, const int dsty, const uint8_t col, const uint8_t bgCol)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'line' should be passed by const reference.](https://app.codacy.com/manual/Raffaello/sdl2-vga-terminal/pullRequest?prid=5403184)